src/mscorlib/shared/System/BitConverter.cs

```diff
@@ -38,6 +48,16 @@ public static byte[] GetBytes(char value)
             return bytes;
         }
 
+        // Converts a char into a Span
+        public static bool TryWriteBytes(Span<byte> destination, char value)
+        {
+            if (destination.Length < Unsafe.SizeOf<char>())
+                return false;
+
+            destination[0] = (byte)value;
```

Comment:
This looks like it's casting the `char` to a `byte` and writing that to the first position in the span, instead of writing the char's two bytes to the span as intended.